examples/lock-app/k32w/README.md

```diff
@@ -21,24 +34,49 @@ board.
 ![K32W061 DK6](../../platform/k32w/doc/images/k32w-dk6.jpg)
 
 The K32W lock example application provides a working demonstration of a
-connected door lock device, built using CHIP, and the NXP K32W061 SDK. The
-example supports remote access (e.g.: using CHIP Tool from a mobile phone) and
-control of a simulated door lock over a low-power, 802.15.4 Thread network. It
-is capable of being paired into an existing CHIP network along with other
-CHIP-enabled devices.
+connected door lock device, built using the Project CHIP codebase and the NXP
+K32W061 SDK. The example supports remote access (e.g.: using CHIP Tool from a
+mobile phone) and control of a simulated door lock over a low-power, 802.15.4
+Thread network. It is capable of being paired into an existing Project CHIP
+network along with other Project CHIP-enabled devices.
 
 The example targets the
 [NXP K32W061 DK6](https://www.nxp.com/products/wireless/thread/k32w061-41-high-performance-secure-and-ultra-low-power-mcu-for-zigbeethread-and-bluetooth-le-5-0-with-built-in-nfc-option:K32W061_41)
 development kit, but is readily adaptable to other K32W-based hardware.
 
-The lock example is intended to serve both as a means to explore the workings of
-CHIP, as well as a template for creating real products based on the NXP
-platform.
+The CHIP device that runs the lock application is controlled by the CHIP
+controller device over the Thread protocol. By default, the CHIP device has
+Thread disabled, and it should be paired over Bluetooth LE with the CHIP
+controller and obtain configuration from it. The actions required before
+establishing full communication are described below.
+
+The example also comes with a test mode, which allows to start Thread with the
+default settings by pressing a button. However, this mode does not guarantee
+that the device will be able to communicate with the CHIP controller and other
+devices.
+
+### Bluetooth LE Advertising
+
+To commission the device onto a Project CHIP network, the device must be
+discoverable over Bluetooth LE. Bluetooth LE advertising is started
+automatically when the device is powered up.
+
+### Bluetooth LE Rendezvous
+
+In Project CHIP, the commissioning procedure (called rendezvous) is done over
```

Comment:
wording here and above seems to imply that BLE is required, however theoretically CHIP devices do not require BLE (Ethernet-only devices or SoftAP). Should we update to say 'in this example' or 'for thread devices'?

Reply:
Right, changed the wording to "In this example".

Reply:
@doru91 would you mind to make this change in the original source as well? Thanks.

Reply:
@LuDuda , sure, operated this changes also in the nrfconnect README.